src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/Marshal.cs

```diff
@@ -824,7 +824,7 @@ public static IntPtr GetHINSTANCE(Module m)
                 case HResults.TYPE_E_TYPEMISMATCH:
                 case HResults.CO_E_NOTINITIALIZED:
                 case HResults.RPC_E_CHANGED_MODE:
```

Comment:
Same here. The whole block of cases can be deleted. The `default:` case is going to take care of this just fine. (github does not allow me to make this as a suggestion.)